app/src/ui/merge-conflicts/merge-conflicts-dialog.tsx

```diff
@@ -232,11 +246,36 @@ export class MergeConflictsDialog extends React.Component<
     return <h3 className="summary">{message}</h3>
   }
 
+  private renderContent(
+    unmergedFiles: ReadonlyArray<WorkingDirectoryFileChange>,
+    conflictedFilesCount: number
+  ): JSX.Element {
+    if (unmergedFiles.length === 0) {
+      return (
```

Comment:
non-blocking:
would prefer to see this JSX rendering as its own named function

Reply:
2cae862